modules/test/playwright/tests/commerce/utils/date.ts

```diff
@@ -74,22 +83,26 @@ export function checkSameDate(dateFromApi: string, date2: string) {
 	const firstDate = new Date(dateFromApi);
 	const secondDate = new Date(date2);
 
-	return firstDate.getUTCDate() === secondDate.getDate() &&
-		   firstDate.getUTCMonth() === secondDate.getMonth() &&
-		   firstDate.getUTCFullYear() === secondDate.getFullYear() &&
-		   firstDate.getUTCHours() === secondDate.getHours() &&
-		   firstDate.getUTCMinutes() === secondDate.getMinutes();
+	return (
+		firstDate.getUTCDate() === secondDate.getDate() &&
+		firstDate.getUTCMonth() === secondDate.getMonth() &&
+		firstDate.getUTCFullYear() === secondDate.getFullYear() &&
+		firstDate.getUTCHours() === secondDate.getHours() &&
+		firstDate.getUTCMinutes() === secondDate.getMinutes()
+	);
 }
 
 export function checkLocalizedDate(dateFromApi: string, date2: string) {
 	const firstDate = new Date(dateFromApi);
 	const secondDate = new Date(date2);
 
-	return firstDate.getDate() === secondDate.getDate() &&
-		   firstDate.getMonth() === secondDate.getMonth() &&
-		   firstDate.getFullYear() === secondDate.getFullYear() &&
-		   firstDate.getHours() === secondDate.getHours() &&
-		   firstDate.getMinutes() === secondDate.getMinutes();
+	return (
```

Comment:
`checkLocalizedDate` and `checkSameDate` are doing the exact same thing, save the usage in one case of `UTC` methods. Can't we merge them and have an additional function argument for discern between the 2?